### PR TITLE
Add page about the dashboard plugin - Closes #1071

### LINKED ISSUE #1071: Add reference page about the dashboard plugin
Create a new page with information about the new dashboard plugin in Lisk SDK 5.1.0

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1071

### CODE REVIEW

File: modules/ROOT/pages/references/lisk-framework/dashboard-plugin.adoc [hunk 1]
```diff
@@ -0,0 +1,111 @@
+= Dashboard plugin
+:toc:
+
+`@liskhq/lisk-framework-dashboard-plugin` is a plugin for the Lisk Framework that provides a UI which allows a developer to conveniently communicate with the blockchain application during the development.
```
Comment (ManuGowda) [inline modules/ROOT/pages/references/lisk-framework/dashboard-plugin.adoc]:
```suggestion
`@liskhq/lisk-framework-dashboard-plugin` is a plugin for the Lisk Framework that provides a user interface which enables developers to conveniently view and communicate with the blockchain application during the development.
```

File: modules/ROOT/pages/references/lisk-framework/dashboard-plugin.adoc [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+The dashbobard plugin includes the following features:
+
+* Monitor recent blocks and transactions
+* Get information about accounts
```
Comment (ManuGowda) [inline modules/ROOT/pages/references/lisk-framework/dashboard-plugin.adoc]:
```suggestion
* Show created information about accounts
```

File: modules/ROOT/pages/references/lisk-framework/dashboard-plugin.adoc [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+* Create new accounts
+* Call actions
```
Comment (ManuGowda) [inline modules/ROOT/pages/references/lisk-framework/dashboard-plugin.adoc]:
```suggestion
* Call exposed application actions
```

File: modules/ROOT/pages/references/lisk-framework/dashboard-plugin.adoc [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+* Listen to events
```
Comment (ManuGowda) [inline modules/ROOT/pages/references/lisk-framework/dashboard-plugin.adoc]:
```suggestion
* Listen to exposed application events
```

File: modules/ROOT/pages/references/lisk-framework/dashboard-plugin.adoc [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+|===
+
+== Registering the plugin
+
+How to register the dashboard plugin with the blockchain application.
+
+[source,js]
+----
+const { Application, DashboardPlugin, genesisBlockDevnet, configDevnet } = require('lisk-sdk');
```
Comment (ManuGowda) [inline modules/ROOT/pages/references/lisk-framework/dashboard-plugin.adoc]: we are not exposing `DashboardPlugin` due to its size through sdk, we need to require explicitly and register.

File: modules/ROOT/pages/references/lisk-framework/dashboard-plugin.adoc [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+    console.error('Faced error in application', error);
+    process.exit(1);
+  });
+----
+
+== Plugin configuration
+
+The dashboard plugin can be configured in the configuration of the blockchain application under `plugins.dashboard`.
+
+NOTE: The application will use the default values in case no plugin configuration is provdided.
```
Comment (chris529) [inline modules/ROOT/pages/references/lisk-framework/dashboard-plugin.adoc]:
```suggestion
NOTE: The application will use the default values in the case whereby there is no plugin configuration provided.
```

**Review by chris529 — COMMENTED** [review]
Looks great, just 1 minor point 👍

**Review by chris529 — APPROVED** [review]
LGTM! 👍

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**